l2/contracts/L2ContractHelper.sol

```diff
@@ -42,3 +42,30 @@ library L2ContractHelper {
         return address(uint160(uint256(data)));
     }
 }
+
+struct Transaction {
+    uint256 txType;
+    uint256 from;
+    uint256 to;
```

Comment:
Our system contracts use `uint256` instead of `address` in a lot of places for the sake of preserving the ability of potential move to 32-byte addresses in the future (not any time soon though). If we were always to pass `address` and then start passing `uint256`, it would be a breaking change. 

Though I agree that it is worth thinking twice before introducing additional complexity to the protocol. I can not answer this question right now (need to reassess it within the team). We definitely not change anything for this particular release, but there will be more :) So we can reiterate on it then

Reply:
Thanks for the info, good idea to be forward looking here